
INIT: Cannot find  mnist  data, downloading now...
2022-04-08 13:11:38.541135: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2022-04-08 13:11:39.338988: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1525] Created device /job:localhost/replica:0/task:0/device:GPU:0 with 11433 MB memory:  -> device: 0, name: TITAN X (Pascal), pci bus id: 0000:04:00.0, compute capability: 6.1
Traceback (most recent call last):
  File "RunTest.py", line 224, in <module>
    main(args)
  File "RunTest.py", line 92, in main
    df_train_losses, train_df, test_df, info = sf.init_data(info)
  File "/user/HS223/ad00878/PhD/curriculum_over_time/project/large_models/supporting_functions.py", line 57, in init_data
    os.mkdirs(info.data_path + info.dataset_name)
AttributeError: module 'os' has no attribute 'mkdirs'